libs/utils/energy_model.py

```diff
@@ -1001,6 +1025,16 @@ def read_idle_states(cpu, domain_level):
                    root_power_domain=root_pd,
                    freq_domains=freq_domains)
 
+    @staticmethod
+    def save_em_yaml(em, yaml_path):
+        with open(yaml_path, "w") as f:
```

Comment:
Do we want to encode some sane default name, e.g. "nrg_model.json"? Same goes for the loading, so you can just point it at some directory and if you used the defaults it will just work™

Reply:
TBH, it does not sound like a great idea. We will end up with code expecting that and not propagating the path in the wrapping APIs just because "it is not required" and we end up with the overwritten files nightmare again.

Reply:
Until we have a better view on how this API will actually be used, having defaults does not really make sense.

Reply:
Fair enough